ts/features/payments/bizEventsTransaction/screens/PaymentsTransactionBizEventsPreviewScreen.tsx

```diff
@@ -47,26 +47,11 @@ const PaymentsTransactionBizEventsPreviewScreen = () => {
     }
     try {
       await Share.open({
-        activityItemSources: [
-          {
-            item: {
-              default: {
-                content: `${I18n.t(
-                  "features.payments.transactions.receipt.title"
-                )}.pdf`,
-                type: "url"
-              }
-            },
-            placeholderItem: {
-              content: `${I18n.t(
-                "features.payments.transactions.receipt.title"
-              )}.pdf`,
-              type: "url"
-            }
-          }
-        ],
         type: "application/pdf",
-        url: `${RECEIPT_DOCUMENT_TYPE_PREFIX}${transactionReceiptFile}`
+        url: `${RECEIPT_DOCUMENT_TYPE_PREFIX}${transactionReceiptFile}`,
+        filename: `${I18n.t(
+          "features.payments.transactions.receipt.title"
+        )}.pdf`
```

Comment:
On Android I have a strange behavior, the file name is "Ricevuta pagoPA.pdf.pdf", it seems to add the file extension automatically. This should help:

```suggestion
          filename: `${I18n.t("features.payments.transactions.receipt.title")}${
          Platform.OS === "ios" ? ".pdf" : ""
        }`
```

I tested on a Rdmi phone, so I don't know if it's something related to the OS.

Reply:
Confirmed: I get this problems on different Android devices. The change suggested above solves the problem!

Reply:
That's perfect! Thank you very much for helping me to solve this issue 🙏

Addressed into 5032c80f2a89c4a3b7b1f5cc677c25519048f278